docs/TROUBLESHOOTING.md

```diff
@@ -214,13 +214,13 @@ This section includes how to debug a failed restore. For more specific issues re
 
 <hr style="height:1px;border:none;color:#333;"> 
 
-<h3 align="center">Issue with Backup/Restore of DeploymentConfig using Restic or restore hooks<a id="deployconfig"></a></h3>
+<h3 align="center">Issue with Backup/Restore of DeploymentConfig using Restic/Kopia or restore hooks<a id="deployconfig"></a></h3>
```

Comment:
For 1.3, this also applies to datamover backups. Based on the updated logic in the controller, I would use this text:
"Issue with Backup/Restore of DeploymentConfig with volumes or restore hooks"

Reply:
Changed.